Changelog.md

```diff
@@ -23,6 +23,7 @@
 - [Allow GDT to be loaded with shared reference](https://github.com/rust-osdev/x86_64/pull/381)
 - [seal off the `PageSize` trait](https://github.com/rust-osdev/x86_64/pull/404)
 - [idt: Fixup Options structure and cleanup set_handler_fn](https://github.com/rust-osdev/x86_64/pull/226)
+- [add `Mapper::clear` to clear any page table entry regardless of present flag](https://github.com/rust-osdev/x86_64/pull/484)
```

Comment:
0.15 has already been released. The `next` branch is currently stale/behind master. I've opened #486 to resolve this. Once #486 is merged, there will be conflicts. I apologize for the delays.

Reply:
So i guess i should leave the changelog unchanged for now?

Reply:
#486 was just merged. If you want to you can rebase onto the latest `next` branch and readd the entry there. If not we'll do it when preparing the next breaking release.

Reply:
I'll do the rebase later when I have some more time. Shouldn't be too much work, but I'm not at a computer with the repo checked out right now.

Reply:
I hope this is correct now. I added a "breaking change" section under the unreleased header.